ksp_plugin_adapter/gl_lines.cs

```diff
@@ -90,19 +96,28 @@ internal static class GLLines {
   }
 
   public static void RenderAndDeleteTrajectory(IntPtr trajectory_iterator,
-                                               UnityEngine.Color colour) {
+                                               UnityEngine.Color colour,
+                                               Style style) {
     try {
       Vector3d? previous_point = null;
 
       UnityEngine.GL.Color(colour);
+      int size = trajectory_iterator.IteratorSize();
 
-      for (; !trajectory_iterator.IteratorAtEnd();
-           trajectory_iterator.IteratorIncrement()) {
+      for (int i = 0;
+           !trajectory_iterator.IteratorAtEnd();
+           trajectory_iterator.IteratorIncrement(), ++i) {
         Vector3d current_point = (Vector3d)trajectory_iterator.IteratorGetXYZ();
         if (previous_point.HasValue) {
-          AddSegment(previous_point.Value,
-                     current_point,
-                     hide_behind_bodies : true);
+          if (style == Style.FADED) {
+            colour.a = (float)i / (float)size;
```

Comment:
Could we not start at 0 but maybe at 20% or some such?  Otherwise the beginning is just invisible which is not useful.